dashboards/redis-otel/redis-otel.json

```
@@ -0,0 +1,612 @@
+{
+  "name": "Redis OTel Quickstart",
+  "description": null,
+  "pages": [
+    {
+      "name": "Redis OTel Proposal",
```

```suggestion
      "name": "Redis OTel",
```

Can I know the reason of Proposal?